tests/e2e/manifests/verify_manifest_lists.go

```diff
@@ -641,7 +645,7 @@ func filterVersions(versions VersionList) (VersionList, error) {
 		minEstimated = version.MustParseGeneric(semVersion)
 		// else get the last version from the previous Major release and apply the skew.
 	} else {
-		url := fmt.Sprintf("https://storage.googleapis.com/kubernetes-release/release/stable-%d.txt", minEstimated.Major()-1)
+		url := fmt.Sprintf("https://dl.k8s.io/release/stable-%d.txt", minEstimated.Major()-1)
```

Comment:
i was going to recommended watching the e2e test when this pr merges, but looks like it's broken for other reasons

https://k8s-testgrid.appspot.com/sig-cluster-lifecycle-all#periodic-manifest-lists

Reply:
added
https://github.com/kubernetes/kubeadm/issues/2873

you can still check if we introduce more errors in the e2e logs ala "URL not available"

Reply:
Looks due to the `arm` removal (possibly) from https://github.com/kubernetes/kubernetes/issues/115738
 
Do you want me to update https://github.com/kubernetes/kubeadm/blob/main/tests/e2e/manifests/verify_manifest_lists.go#L105 with this PR?

Reply:
a separate PR that refs #2873 is better, if you can help us it would be great! note it must version check for > 1.26.x and not look for arm in those versions.

another test infra PR is needed to send alerts to
`testgrid-alert-email: sig-cluster-lifecycle-kubeadm-alerts@kubernetes.io`

and replace the alert dest here
https://prow.k8s.io/prowjob?prowjob=e031104e-59df-41d8-9afc-34543984fa1f